website/docs/r/evidently_launch.html.markdown

```diff
@@ -0,0 +1,349 @@
+
+  groups {
+    feature   = aws_evidently_feature.example.name
+    name      = "Variation1"
+    variation = "Variation1"
+  }
+
+  groups {
+    feature   = aws_evidently_feature.example.name
+    name      = "Variation2"
+    variation = "Variation2"
+  }
+
+  scheduled_splits_config {
+    steps {
+      group_weights = {
+        "Variation1" = 0
+        "Variation2" = 0
+      }
+
+      segment_overrides {
+        evaluation_order = 1
+        segment          = aws_evidently_segment.example.name
+
+        weights = {
+          "Variation2" = 10000
+        }
+      }
+
+      segment_overrides {
+        evaluation_order = 2
+        segment          = aws_evidently_segment.example.name
+
+        weights = {
+          "Variation1" = 40000
+          "Variation2" = 30000
+        }
+      }
+
+      start_time = "2024-01-08 01:43:59+00:00"
+    }
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `description` - (Optional) Specifies the description of the launch.
+* `groups` - (Required) One or up to five blocks that contain the feature and variations that are to be used for the launch. [Detailed below](#groups).
+* `metric_monitors` - (Optional) One or up to three blocks that define the metrics that will be used to monitor the launch performance. [Detailed below](#metric_monitors).
+* `name` - (Required) The name for the new launch. Minimum length of `1`. Maximum length of `127`.
+* `project` - (Required) The name or ARN of the project that is to contain the new launch.
+* `randomization_salt` - (Optional) When Evidently assigns a particular user session to a launch, it must use a randomization ID to determine which variation the user session is served. This randomization ID is a combination of the entity ID and randomizationSalt. If you omit randomizationSalt, Evidently uses the launch name as the randomizationSalt.
+* `scheduled_splits_config` - (Optional) A block that defines the traffic allocation percentages among the feature variations during each step of the launch. [Detailed below](#scheduled_splits_config).
+* `tags` - (Optional) Tags to apply to the launch. If configured with a provider [`default_tags` configuration block](/docs/providers/aws/index.html#default_tags-configuration-block) present, tags with matching keys will overwrite those defined at the provider-level.
+
+### `groups`
+
+The `groups` block supports the following arguments:
+
+* `description` - (Optional) Specifies the description of the launch group.
+* `feature` - (Required) Specifies the name of the feature that the launch is using.
+* `name` - (Required) Specifies the name of the lahnch group.
+* `variation` - (Required) Specifies the feature variation to use for this launch group.
+
+### `metric_monitors`
+
+The `metric_monitors` block supports the following arguments:
+
+* `metric_definition` - (Required) A block that defines the metric. [Detailed below](#metric_definition).
```

Comment:
```suggestion
* `metric_definition` - (Required) A block that defines the metric. [Detailed below](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/evidently_launch#metric_definition).
```

Reply:
Following the Kendra docs https://github.com/hashicorp/terraform-provider-aws/blob/fcb0d1ed516f6036c145db8f1ea76709c2298382/website/docs/r/kendra_experience.html.markdown?plain=1#L49 full length URLs are not required for linking, pretty cool stuff